src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -375,389 +299,12 @@ private SuggestedActionSet InlineActions(SuggestedActionSet actionSet)
-                    // a result category which is the maximum severity category amongst the set
-                    var minStart = -1;
-                    var maxEnd = -1;
-                    var category = PredefinedSuggestedActionCategoryNames.CodeFix;
-
-                    foreach (var set in sets)
-                    {
-                        if (set.ApplicableToSpan.HasValue)
-                        {
-                            var currentStart = set.ApplicableToSpan.Value.Start;
-                            var currentEnd = set.ApplicableToSpan.Value.End;
-
-                            if (minStart == -1 || currentStart < minStart)
-                            {
-                                minStart = currentStart;
-                            }
-
-                            if (maxEnd == -1 || currentEnd > maxEnd)
-                            {
-                                maxEnd = currentEnd;
-                            }
-                        }
-
-                        Debug.Assert(set.CategoryName == PredefinedSuggestedActionCategoryNames.CodeFix ||
-                                     set.CategoryName == PredefinedSuggestedActionCategoryNames.ErrorFix);
-
-                        // If this set contains an error fix, then change the result category to ErrorFix
-                        if (set.CategoryName == PredefinedSuggestedActionCategoryNames.ErrorFix)
-                        {
-                            category = PredefinedSuggestedActionCategoryNames.ErrorFix;
-                        }
-                    }
-
-                    var combinedSpan = minStart >= 0 ? new Span(minStart, maxEnd) : (Span?)null;
-                    return (combinedSpan, category);
+                    return UnifiedSuggestedActionsSource.GetFilterAndOrderCodeFixes_MustBeCalledFromUIThread(
+                        workspace, _owner._codeFixService, document, range.Span.ToTextSpan(),
+                        includeSuppressionFixes, isBlocking: true, addOperationScope, cancellationToken);
                 }
-            }
 
-            private static void AddSuggestedActionsSet(
-                IEnumerable<SuggestedAction> actions,
-                CodeFixGroupKey groupKey,
-                ArrayBuilder<SuggestedActionSet> sets)
-            {
-                foreach (var group in actions.GroupBy(a => a.Priority))
-                {
-                    var priority = GetSuggestedActionSetPriority(group.Key);
-
-                    // diagnostic from things like build shouldn't reach here since we don't support LB for those diagnostics
-                    Debug.Assert(groupKey.Item1.HasTextSpan);
-                    var category = GetFixCategory(groupKey.Item1.Severity);
-                    sets.Add(new SuggestedActionSet(category, group, priority: priority, applicableToSpan: groupKey.Item1.GetTextSpan().ToSpan()));
-                }
+                return ImmutableArray<UnifiedSuggestedActionSet>.Empty;
```

Comment:
nit: you don't have to change this.  but i far prefer:

```
if (test)
   return empty;

// do long stuff
```

versus

```
if test
{
     // long stuff
}

return empty;
```

It took me a while to figure out waht this condition this branch was associated with.